config/environments/production.rb

```diff
@@ -64,6 +64,13 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.delivery_method = :ses
+
+  config.action_mailer.default_url_options = {
+    protocol: 'https',
+    host: 'gov-au-beta.apps.platform.digital.gov.au'
```

Comment:
@freshtonic you can access them via `VCAP_APPLICATION` environment variable which should contain JSON we need to parse (https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#app-system-env)

We'd need to access `application-uris` and regex out any coloured URLs (-blue and -green variants). The remaining uri would be the one we want to keep. 

**note** during our deploy, we're likely to remove the route to our app (which would leave _only_ the coloured variant URI). As such, we'd need to use regex to remove the colour from the URI we have access to.